crates/testing/src/sdk.rs

```diff
@@ -145,16 +145,7 @@ fn compile_module(module: &str) -> String {
 // module as a separate clean database instance for isolation purposes.
 fn publish_module(wasm_file: &str) -> String {
     let name = random_module_name();
-    invoke_cli(&[
-        "publish",
-        "--server",
-        "local",
-        "--debug",
-        "--project-path",
-        wasm_file,
-        "--skip_clippy",
-        &name,
-    ]);
+    invoke_cli(&["publish", "--server", "local", "--wasm-file", wasm_file, &name]);
```

Comment:
it turns out that `--debug` and `--skip_clippy` were essentially unused, because we pass `--wasm-file` anyway, skipping the build step.

It became an error in this PR because I added a `.conflicts_with("build_options")` to `--wasm-file`.

Reply:
(I also changed `--project-path` to `--wasm-file` to make it clearer. That usage of `--project-path` is deprecated anyway.)